common/source/docs/common-cuav-pwlink.rst

```diff
@@ -17,8 +17,14 @@ Connecting to the Ground Station
 ================================
 
 Open WIFI connection dialog on PC or Android Phone and select the ``CUAVWLINKxxxx`` SSID and connect using the password ``cuavwlink``. Once connected use the UDP connection with port = 14550 (which is default) to connect using Mission Planner. QGC should auto-detect and connect.
-The PW-LINK ahs a default IP address of ``192.168.4.1``, which is used for a mavproxy connection.
+The PW-LINK default IP address is ``192.168.4.1``.
```

Comment:
since its a draft I cant suggest....but I would make this line clearer:
he PW-LINK default IP address is ``192.168.4.1`` for its web interface to change communication parameters within the  PW-LINK.

Reply:
Thanks. I put it ready to review for others to add suggestions

Reply:
Yep, applied it.

Reply:
Henry, I tried opening the web page on this address, and it does not show anything. What am I missing?

Reply:
``` 
ryan@B650-970:~$ wget 192.168.4.1
--2024-05-03 17:39:06--  http://192.168.4.1/
Connecting to 192.168.4.1:80... failed: Connection refused.

```

Reply:
I never have used wget file transfer utility...just open the address in a web browser....assuming you have your wifi connected to the PWLink hotspot

Reply:
Yea, I tried in web first. It didn't connect and show anything.

Reply:
sorry, been awhile...the CUAV-PWlink does not have the web interface...flashing MAVEsp8266 to it provides the web interface https://github.com/ArduPilot/mavesp8266?tab=readme-ov-file

Reply:
Ok, I'll drop that change off. This page doesn't have anything to do with MAVEsp8266.